pyfenix/__main__.py

```diff
@@ -1,16 +1,28 @@
 """Fenix runner"""
 
-import queue
+import asyncio
+from queue import Queue
 import threading
 
-from tests.doubles import ClientAPIStub, GUIEventHandlerStub
+from tests.doubles import GUIEventHandlerStub
 
 from .client_event_loop import ClientEventLoop
+from .ws_api import WebsocketsAPI
 
-event_queue: queue.Queue = queue.Queue()
-api = ClientAPIStub(event_queue)
-gui = GUIEventHandlerStub(event_queue)
+async def main() -> None:
+    """Run Fenix"""
+    event_queue: Queue = Queue()
+    api = WebsocketsAPI(event_queue)
+    await api.connect(("localhost", 60221))
+    await api.send("yay")
```

Comment:
What is this for?

Reply:
I was testing the components together manually to ensure it works with a real server.